README.md

```diff
@@ -151,108 +153,107 @@ To find the unique identifier of an extension:
 - Navigate to the extension's web page, for example [https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter&WT.mc_id=academic-77460-alfredodeza)
 - Locate the *Unique Identifier* field under **More info** section on your right side.
 
-</details>
+## Deploy the API app
 
-<details> 
-
-<summary><b>How to deploy</b></summary>
-
-Now, we are going to deploy our application using Azure and GitHub actions to do this autmomatically! However, we need to configure our services.
+Now, you are going to deploy the application using Azure and GitHub actions to do this autmomatically! However, you need to configure some services first.
 
 ### Azure Account
 
-First, using free resources with an Azure subscription. Using one of these to deploy it.
+This app can be deployed entirely using free resources on a free Azure account. Sign in or sign up with one of these links:
 
-- [Sign in to your account](https://azure.microsoft.com/en-US/?WT.mc_id=academic-77460-alfredodeza)
+- [Sign in to your Azure account](https://azure.microsoft.com/en-US/?WT.mc_id=academic-77460-alfredodeza)
 - [Create a (no Credit Card required) Azure For Students account](https://azure.microsoft.com/free/students/?WT.mc_id=academic-77460-alfredodeza)
 - [Create a new Azure account](https://azure.microsoft.com/en-US/?WT.mc_id=academic-77460-alfredodeza)
 
 ### Create an Azure App Service
 
-Now, let's create our Azure resources.
+Now, you need to create Azure "resources" using the `az` command-line tool, which you can do entirely in the browser.
 
-- Open an [Azure Cloud Shell](https://shell.azure.com/?WT.mc_id=academic-77460-alfredodeza) to use the `az` cli. 
-- Use Bash for this guide to generate your unique identifier quickly.
-- Create a *Resource Group*:
+- Open the [Azure Cloud Shell](https://shell.azure.com/?WT.mc_id=academic-77460-alfredodeza).
+- If it asks to choose between _Bash_ and _Powershell_ environments, select _Bash_.
+- If it says "You have no storage mounted", select a subscription in your account and click "Create storage". The Cloud Shell uses that storage resource to store data generated during your shell sessions.
+- Create a *Resource Group* which will group together the different Azure resources used for the app:
 ```
 az group create --name demo-fastapi --location "East US"
 ```
-- Create the **FREE** App Service Plan:
+- You'll see a JSON response with details about the newly created resource, for this command and all the commands that follow.
+- Create the **FREE** *App Service Plan*:
```

Comment:
If the App Service Plan is FREE, does that imply the Web App Service is not free?